### ISSUE TITLE
default photo for thumbnail?

### ISSUE THREAD

**Issue by sheppard** [Soil-Carbon-Coalition/atlasbiowork#25]
It would be nice to have a thumbnail to use in observation lists and map popups.  One way to do this would be through a custom `ObservationSerializer` field that would scan the JSON data for photo references and return the first one.

**Comment by managingwholes** [comment]
Yes. For the map browser (more usable on larger screens) I'm not planning on using popups: just 2 or 3 fields for hover, and on click the whole parsed object will go into an expandable sidebar, and for this I would want more than the 160px thumbnails but perhaps the 1024 size.

I want to learn how to do serializers, and wonder for down the road if it might be possible to load data in response to a geographic query, e.g. features within a bounding box, and query differences over time between observations.
